### ISSUE TITLE
Target page link fails aria-valid-attr-value

### ISSUE THREAD

**Issue by lisli1** [microsoft/accessibility-insights-web#1904]
## Describe the bug

**To Reproduce**

1. Go to https://www.washington.edu/accesscomputing/AU/before.html
2. Open the extension and run FastPass or Assessment
3. Open browser console (right-click details view page > Inspect > Console tab)
4. Run `a11ySelfValidator.validate()` in the console
5. Expand result to see error:
```
id: "aria-valid-attr-value"
description: "Ensures all ARIA attributes have valid values"
nodes: Array(1)
0: {domElement: button.ms-Link.insights-link.target-page-link.root-74, all: Array(1), any: Array(0), none: Array(0), target: Array(1)
```

## Expected behavior

The `a11ySelfValidator` should return an empty array (no errors)

## Context (please complete the following information)

- OS Name & Version:  Windows 10.0.18363 Build 18363
- AI-Web Version & Environment:  Canary 2019.12.20.2204
- Browser Version:  Google Chrome Version 79.0.3945.88 (Official Build) (64-bit)
- Target Page: https://www.washington.edu/accesscomputing/AU/before.html

## Are you willing to submit a PR?

Yes

## Did you search for similar existing issues?

No
